test/e2e/agent/agent_updates_test.go

```diff
@@ -27,7 +27,7 @@ var _ = Describe("VM Agent behavior during updates", func() {
 	})
 
 	Context("updates", func() {
-		It("should update to the requested image", Label("updates"), func() {
+		It("should update to the requested image", Label("updates", "75523"), func() {
```

Comment:
```suggestion
		It("should update to the requested image", Label("updates", "75523"), func() {
```

Can we add some prefix to 75523, like rh-75523 or redhat-75523, the plan is that this will potentially be an upstream product with other collaborators, and the number would be meaningless to them :), may be somebody would eventually remove it if it's not clear.

Reply:
Done,changed to rh-